lib/skate_web/endpoint.ex

```diff
@@ -1,46 +1,50 @@
 defmodule SkateWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :skate
 
-  socket "/socket", SkateWeb.UserSocket,
+  socket("/socket", SkateWeb.UserSocket,
     websocket: [check_origin: Application.get_env(:skate, :websocket_check_origin, false)],
     longpoll: false
+  )
 
   # Serve at "/" the static files from "priv/static" directory.
   #
   # You should set gzip to true if you are running phx.digest
   # when deploying your static files in production.
-  plug Plug.Static,
+  plug(Plug.Static,
     at: "/",
     from: :skate,
     gzip: false,
     only: ~w(css fonts images js favicon.ico robots.txt)
+  )
 
   # Code reloading can be explicitly enabled under the
   # :code_reloader configuration of your endpoint.
   if code_reloading? do
-    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
-    plug Phoenix.LiveReloader
-    plug Phoenix.CodeReloader
+    socket("/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket)
+    plug(Phoenix.LiveReloader)
+    plug(Phoenix.CodeReloader)
   end
 
-  plug Plug.RequestId
-  plug Plug.Logger
+  plug(Plug.RequestId)
+  plug(Plug.Logger)
 
-  plug Plug.Parsers,
+  plug(Plug.Parsers,
     parsers: [:urlencoded, :multipart, :json],
     pass: ["*/*"],
     json_decoder: Phoenix.json_library()
+  )
 
-  plug Plug.MethodOverride
-  plug Plug.Head
+  plug(Plug.MethodOverride)
+  plug(Plug.Head)
 
   # The session will be stored in the cookie and signed,
   # this means its contents can be read but not tampered with.
   # Set :encryption_salt if you would also like to encrypt it.
-  plug Plug.Session,
+  plug(Plug.Session,
     store: :cookie,
     key: "_skate_key",
     signing_salt: "jkUgGkwy"
+  )
 
-  plug SkateWeb.Router
+  plug(SkateWeb.Router)
```

Comment:
I noticed you got rid of the `(` `)` formatting in `router.ex`. Did you also want to revert the formatting here?

Reply:
Ok